bot/cogs/tags.py

```diff
@@ -36,18 +36,32 @@ def get_tags() -> dict:
         """Get all tags."""
         # Save all tags in memory.
         cache = {}
-        tag_files = Path("bot", "resources", "tags").iterdir()
+        tag_files = Path("bot", "resources", "tags").glob("**/*")
         for file in tag_files:
-            tag_title = file.stem
-            tag = {
-                "title": tag_title,
-                "embed": {
-                    "description": file.read_text()
+            file_path = str(file).split("/")
+            if file.is_file():
+                tag_title = file.stem
+                tag = {
+                    "title": tag_title,
+                    "embed": {
+                        "description": file.read_text()
+                    },
+                    "restricted_to": "developers"
                 }
-            }
-            cache[tag_title] = tag
+                if len(file_path) == 5:
+                    restricted_to = file_path[3]
+                    tag["restricted_to"] = restricted_to
```

Comment:
Why is this required? Isn't easier to make `tag["restricted_to"] = file_path[3]`?